modules/internal/ChapelArray.chpl

```diff
@@ -1383,7 +1383,7 @@ module ChapelArray {
     // checks should be performed and is set based on the value of
     // the --no-formal-domain-checks flag.
     //
-    inline proc chpl_checkArrArgDoms(formalDom: domain, param runtimeChecks: bool) {
+    proc chpl_checkArrArgDoms(formalDom: domain, param runtimeChecks: bool) {
```

Comment:
This wasn't private before, and `chpl_` does that to some extent. But if its usage is limited to this module, consider `private`.